presto-parser/src/main/java/com/facebook/presto/sql/tree/RenameTable.java

```diff
@@ -42,6 +42,12 @@ private RenameTable(Optional<NodeLocation> location, QualifiedName source, Quali
         this.target = requireNonNull(target, "target name is null");
     }
 
+    @Override
+    public QualifiedName getQualifiedName()
```

Comment:
What you need for beginQuery is the connector right? I see how getQualifiedName is the cleanest way to do that, but I suppose you could return the connector id itself.